src/parser/CarsParser.java

```diff
@@ -0,0 +1,39 @@
+package parser;
+
+import entity.Car;
+import entity.Company;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CarsParser {
+    private static List<Car> cars = new ArrayList<>();
+    private static final Logger LOGGER = Logger.getLogger(Company.class.getName());
+    private static final AttributeValueParser attributeValueParser = new AttributeValueParser();
+
+    public List<Car> getCars(final List<String> carList) {
+        Pattern pattern = Pattern.compile("\\d+");
+
+        for (final String fileLine : carList) {
+            String[] splitedText = fileLine.split(",");
```

Comment:
Make final. If you don't need to reassign value for an reference then you need to make a variable as final.